case-lib/lib.sh

```diff
@@ -296,9 +296,15 @@ fake_kern_error()
 get_ldc_subdir()
 {
     local subdir='intel/sof' # default
-    if test -e /sys/module/snd_sof_pci/parameters/fw_path; then
-        local fw_path
-        fw_path=$(cat /sys/module/snd_sof_pci/parameters/fw_path)
+    local fw_path
+    local fw_path_info='/sys/kernel/debug/sof/fw_path'
+
+    if test ! -e $fw_path_info; then
+	fw_path_info='/sys/module/snd_sof_pci/parameters/fw_path'
```

Comment:
Nit: the shorter, "idiomatic" style is:
```shell
test -e $fw_path_info ||
    fw_path_info='/sys/module/snd_sof_pci/parameters/fw_path'
```

It reads in plain English likes this: "either $fw_path_info exists, OR we redefine $fw_path_info and point it at the alternative"

Reply:
done